data/src/main/java/io/forus/me/android/data/repository/vouchers/VouchersRepository.kt

```diff
@@ -116,12 +115,40 @@ class VouchersRepository(private val vouchersDataSource: VouchersDataSource) : i
         return VoucherProvider(item, organizations, categories)
     }
 
+
+    private fun getFakeVoucherProvider(): VoucherProvider {
+        val date = Calendar.getInstance().getTime()
+        val currency = Currency("EUR", "transaction.currency?.logoUrl")
+
+        val organization = Organization(0, "Fly Avia", "logo", 0.0, 0.0,
```

Comment:
Could we use their own organisation name? logo etc

Reply:
@maxvisser there is no provider on the test transaction (no api)